validation of example bundles on azure pipelines fails

```
Generating script.
Script contents:
make build install build-bundle validate-bundle
========================== Starting Command Output ===========================
[command]/bin/bash --noprofile --norc /home/vsts/work/_temp/c0c43710-0f07-4723-9557-f286fff64721.sh
go generate ./...
make --no-print-directory build MIXIN=porter -f mixin.mk BINDIR=bin
mkdir -p bin
go build -ldflags '-w -X github.com/deislabs/porter/pkg.Version=v0.10.0-beta.2-17-gf2a1e5c -X github.com/deislabs/porter/pkg.Commit=f2a1e5c' -o bin/porter ./cmd/porter
mkdir -p bin
GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -a -tags netgo -ldflags '-w -X github.com/deislabs/porter/pkg.Version=v0.10.0-beta.2-17-gf2a1e5c -X github.com/deislabs/porter/pkg.Commit=f2a1e5c' -o bin/porter-runtime ./cmd/porter
make --no-print-directory build MIXIN=exec -f mixin.mk
mkdir -p bin/mixins/exec
go build -ldflags '-w -X github.com/deislabs/porter/pkg.Version=v0.10.0-beta.2-17-gf2a1e5c -X github.com/deislabs/porter/pkg.Commit=f2a1e5c' -o bin/mixins/exec/exec ./cmd/exec
mkdir -p bin/mixins/exec
GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -a -tags netgo -ldflags '-w -X github.com/deislabs/porter/pkg.Version=v0.10.0-beta.2-17-gf2a1e5c -X github.com/deislabs/porter/pkg.Commit=f2a1e5c' -o bin/mixins/exec/exec-runtime ./cmd/exec
make --no-print-directory build MIXIN=kubernetes -f mixin.mk
mkdir -p bin/mixins/kubernetes
go build -ldflags '-w -X github.com/deislabs/porter/pkg.Version=v0.10.0-beta.2-17-gf2a1e5c -X github.com/deislabs/porter/pkg.Commit=f2a1e5c' -o bin/mixins/kubernetes/kubernetes ./cmd/kubernetes
mkdir -p bin/mixins/kubernetes
GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -a -tags netgo -ldflags '-w -X github.com/deislabs/porter/pkg.Version=v0.10.0-beta.2-17-gf2a1e5c -X github.com/deislabs/porter/pkg.Commit=f2a1e5c' -o bin/mixins/kubernetes/kubernetes-runtime ./cmd/kubernetes
cd pkg/porter && packr2 clean
`cd pkg/exec && packr2 clean`;   `cd pkg/kubernetes && packr2 clean`; 
bin/porter mixin install helm --version canary --url https://cdn.deislabs.io/porter/mixins/helm;   bin/porter mixin install azure --version canary --url https://cdn.deislabs.io/porter/mixins/azure;   bin/porter mixin install terraform --version canary --url https://cdn.deislabs.io/porter/mixins/terraform; 
installed helm mixin
helm mixin v0.6.0-beta.1-4-g88e07ec (88e07ec)
installed azure mixin
Azure mixin v0.6.1-beta.1-7-gb0a41fd (b0a41fd)
installed terraform mixin
terraform mixin v0.2.1-beta.1-4-g0d24b85 (0d24b85)
mkdir -p /home/vsts/.porter
cp -R bin/mixins /home/vsts/.porter/
cp bin/porter* /home/vsts/.porter/
ln -f -s /home/vsts/.porter/porter /usr/local/bin/porter
ln: failed to create symbolic link '/usr/local/bin/porter': Permission denied
Makefile:172: recipe for target 'install' failed
make: *** [install] Error 1
##[error]Bash exited with code '2'.
##[section]Finishing: Validate Example Bundles
```

--- a/Makefile
+++ b/Makefile
@@ -148,7 +148,7 @@ build-bundle:
 ifndef BUNDLE
 	$(call all-bundles,$(EXAMPLES_DIR),build-bundle)
 else
-	cd $(EXAMPLES_DIR)/$(BUNDLE) && porter build
+	cd $(EXAMPLES_DIR)/$(BUNDLE) && ../../bin/porter build
 endif
 
 .PHONY: publish-bundle
@@ -156,7 +156,7 @@ publish-bundle:
 ifndef BUNDLE
 	$(call all-bundles,$(EXAMPLES_DIR),publish-bundle)
 else
-	cd $(EXAMPLES_DIR)/$(BUNDLE) && porter publish
+	cd $(EXAMPLES_DIR)/$(BUNDLE) && ../../bin/porter publish
 endif
 
 BUNDLE_SCHEMA      := bundle.schema.json
@@ -175,8 +175,14 @@ fetch-bundle-schema:
 fetch-definitions-schema:
 	$(call fetch-schema,$(DEFINITIONS_SCHEMA))
 
+HAS_AJV := $(shell command -v ajv)
+ajv:
+ifndef HAS_AJV
+	npm install -g ajv-cli
+endif
+
 .PHONY: validate-bundle
-validate-bundle: fetch-schemas
+validate-bundle: fetch-schemas ajv
 ifndef BUNDLE
 	$(call all-bundles,$(EXAMPLES_DIR),validate-bundle)
 else

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -51,14 +51,11 @@ steps:
   workingDirectory: '$(modulePath)'
   displayName: 'Cross Compile'
 
-# TODO: add example bundle validation
-# Currently failing on installing porter (via make install)
-# See https://dev.azure.com/deislabs/porter/_build/results?buildId=2608
-#
-# - script: |
-#      make build install build-bundle validate-bundle
-#   workingDirectory: '$(modulePath)'
-#   displayName: 'Validate Example Bundles'
+- script: |
+    sudo make ajv
+    make build-bundle validate-bundle
+  workingDirectory: '$(modulePath)'
+  displayName: 'Validate Example Bundles'
 
 - task: DownloadSecureFile@1
   inputs:

